modules/cnf-image-based-upgrade-prep-resources.adoc

```diff
@@ -0,0 +1,431 @@
+// Module included in the following assemblies:
+// * scalability_and_performance/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cnf-image-based-upgrade-creating-backup-resources_{context}"]
+= Creating ConfigMap objects for the image-based upgrade with {lcao}
+
+The {lcao} needs all your OADP resources, extra manifests, and custom catalog sources wrapped in a `ConfigMap` object to process them for the image-based upgrade.
+
+
+[id="cnf-image-based-upgrade-creating-backup-oadp-resources_{context}"]
+== Creating OADP ConfigMap objects for the image-based upgrade with {lcao}
+
```

Comment:
See "IBU Docs Additions" - `Guidelines for application backup/restore CRs`.  I think we want a new section to describe more details for the backup/restore. Suggest to add application backup/restore CRs in the new section and they can be referenced in both non-gitops and gitops way for oadp configmap creation.